### PR TITLE
2.0.0

### PR DESCRIPTION
**Pull request by iamtesch**
Combined changelog:

hebiros

2.0.0 (2019-01-30)
------------------
* *Breaking change: changed AddGroupFromUrdf service to AddGroupFromURDF*
* Restructuring of internal code
* Addition of simulator backing support
* Checks for existing group before making a new group
* Add support for making RobotModel from URDF for IK support
* Added FK support
* Updated dependency on core C++ API

hebiros_advanced_examples

2.0.0 (2019-01-30)
-------------------
* Added omnibase examples

hebiros_basic_examples

2.0.0 (2019-01-30)
------------------
* Added forward kinematics example

hebiros_description

2.0.0 (2019-01-30)
------------------
* Added more hebi components as Xacro macros.

hebiros_gazebo_plugin

2.0.0 (2019-01-30)
------------------
* Initial release of gazebo plugin for HEBI
  components; still in beta stage.

### CODE REVIEW

File: hebiros/CMakeLists.txt [hunk 1]
```diff
@@ -242,10 +256,9 @@ target_link_libraries(hebiros_node ${catkin_LIBRARIES} ${PROJECT_SOURCE_DIR}/lib
 #############
 
 ## Add gtest based cpp test target and link libraries
-# catkin_add_gtest(${PROJECT_NAME}-test test/test_hebiros.cpp)
-# if(TARGET ${PROJECT_NAME}-test)
-#   target_link_libraries(${PROJECT_NAME}-test ${PROJECT_NAME})
-# endif()
+find_package(rostest REQUIRED)
```
Comment (rLinks234) [inline hebiros/CMakeLists.txt]: Is this where `gtest` was getting included from?

File: hebiros/include/hebiros/hebiros.h [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+#include "hebiros_parameters.h"
+#include "hebiros_model.h"
+
+#include <iostream>
+#include <chrono>
+#include <thread>
+#include <mutex>
+#include <boost/bind.hpp>
+#include <execinfo.h>
+#include <signal.h>
```
Comment (rLinks234) [inline hebiros/include/hebiros/hebiros.h]: What are we doing with signals...?

File: hebiros/include/hebiros/hebiros_actions.h [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+#include "hebiros/TrajectoryAction.h"
+
+
+class HebirosActions {
+
+  public:
+
+    static std::map<std::string,
+      std::shared_ptr<actionlib::SimpleActionServer<hebiros::TrajectoryAction>>>
```
Comment (rLinks234) [inline hebiros/include/hebiros/hebiros_actions.h]: :+1: :heart: :heart: :heart: :heart: for not `> > >`

File: hebiros/CMakeLists.txt (hunk 1 above)
Comment (iamtesch) [reply hebiros/CMakeLists.txt]: Nope -- that was on a completely different repository.  This is for the unit tests we have set up here.

**Pull request merged**